includes/sqp/1-workflow.md

```diff
@@ -4,28 +4,12 @@
 
     **Depending what's released first and available the following Workflow Logic will be used:**
 
-    - When a HD WEBDL is released it will download the HD WEB-DL. (streaming services)
+    - When a 1080p WEBDL is released it will download the 1080p WEB-DL. (streaming services)
     - When a BHDStudio is released it will download the BHDStudio releases.
-    - If no BHDStudio can be found (not existing or limited sources) it will prefer HQ WEB-DL.
+    - If no BHDStudio can be found (not existing or limited sources) it will download a 1080p Bluray Encodes (less or not streaming optimized).
```

Comment:
If no BHDStudio or WEBDL? It would still prefer a WEBDL to an encode?

Reply:
no it's the workflow order
![image](https://github.com/TRaSH-Guides/Guides/assets/6155095/10b51406-3f15-4796-99b2-5059593b1820)
so if there is NO WEBDL and NO BHDStudio, it will pick up a HD Encode

Reply:
Cool, I'll leave this as it is.

Reply:
Retained existing.